data/base-stealing.yaml

```diff
@@ -5454,9 +5466,11 @@
   acceptable_min: 284
   poorly_max: 291
   poorly_min: 291
+  vpoorly_max: 387
+  vpoorly_min: 387
 - id: '522'
   item: hip pouch
-  item_in: ' catalog'
+  item_in: " catalog"
```

Comment:
This doesn't look right. Do you know anyone in Shard that could double check it?